src/reducers/navroot/navroot.js

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
These reducers need to write to the ['components'] expanders, if the expanders are defined. See breadcrumbs for more info. I'm also missing tests for them.

Reply:
I wasn't aware of it. I will check it.